code/particle/ParticleParse.cpp

```diff
@@ -229,12 +229,10 @@ namespace particle {
 				int& curve = output == 0 ? effect.m_size_lifetime_curve : effect.m_vel_lifetime_curve;
 
 				required_string("+Curve Name:");
```

Comment:
How about adding `+Curve:` as an alias?  (Also in other places where `+Curve Name:` is used.)